package-lock.json

```diff
@@ -4383,7 +4383,7 @@
         },
         "htmlparser2": {
           "version": "3.8.3",
-          "resolved": "https://registry.npmjs.org/htmlparser2/-/htmlparser2-3.8.3.tgz",
+          "resolved": "http://registry.npmjs.org/htmlparser2/-/htmlparser2-3.8.3.tgz",
```

Comment:
Hmmm?

Reply:
I have no idea how this happened. @garethtdavies perhaps related to the fact I'm not using the Dockerfile. Changing it back to `https`.

Reply:
I also have no idea. A brief Google search and there seem to be quite a few mentions of it but I haven't researched more than a cursory skim read of something like https://github.com/npm/npm/issues/20106

Reply:
Yep, I don't think it's worth the time tbh.